fashion-messaging-app-api/server.js

```diff
@@ -173,45 +172,103 @@ app.get('/recommendations', async (req, res) => {
       return res.status(401).json({ error: 'Unauthorized' });
     }
 
+    let recommendations = [];  
+    const frequentSearch = await SearchResult.findOne({
+      where: { 
+        userId: req.session.user.id,
+        searchTerm: {
+          [Op.not]: defaultQuery  
+        }
+      },
+      order: [['searchCount', 'DESC']],
```

Comment:
This helps me check the searchCount in depending order, from highest to lowest then recommend based on the highest searchCount